content/ch-states/representing-qubit-states.ipynb

```diff
@@ -152,7 +152,8 @@
    "source": [
     "from qiskit import QuantumCircuit, assemble, Aer\n",
     "from qiskit.visualization import plot_histogram, plot_bloch_vector\n",
-    "from math import sqrt, pi"
+    "from math import sqrt, pi\n",
+    "from qiskit.providers.aer import Aer"
```

Comment:
```suggestion
    "from qiskit_aer import Aer"
```
I think this is best practice now, see [this comment](https://github.com/qiskit-community/qiskit-textbook/pull/1487#issue-1375669013).